doc/Development_Documentation/03_Documents/03_Navigation.md

```diff
@@ -487,6 +487,15 @@ $navigation->addPage([
    'title' => 'Homepage' //tooltip text
 ]);
 ```
+```twig
+ {% set mainNavigation = mainNavigation.addPage({
+                            'order': -1,
+                            'uri': navStartNode.url,
+                            'label': 'Home',
+                            'title': "Home"
+                        })
+%}
```

Comment:
There's no need in re-assigning the variable, so this should be sufficient:

```suggestion
{{ mainNavigation.addPage({
    order: -1, // put it in front of all the others
    uri: '/', //path to homepage
    label: 'Home', //visible label
    title: 'Homepage', //tooltip text
}) }}
```

Reply:
@jdreesen unfortunately not. Your suggested change was my first shot too but this construction ends up with `Object of class Pimcore\Navigation\Container could not be converted to string` error. Therefore the re-assignment.

Reply:
`{% do mainNavigation.addPage.... %}`

Reply:
@dpfaffenbauer you're right, the`do` is the correct  solution here